VCFv4.5.draft.tex

```diff
@@ -476,12 +477,17 @@ \subsubsection{Genotype fields}
       ADR		& R		& Integer	& Read depth for each allele on the reverse strand \\
       DP		& 1		& Integer	& Read depth \\
       EC		& A		& Integer	& Expected alternate allele counts \\
+      END		& 1		& Integer	& End position on CHROM (used with multi-sample $<$*$>$ alleles) \\
       FT		& 1		& String	& Filter indicating if this genotype was ``called'' \\
       GL		& G		& Float		& Genotype likelihoods \\
       GP		& G		& Float		& Genotype posterior probabilities \\
       GQ		& 1		& Integer	& Conditional genotype quality \\
       GT		& 1		& String	& Genotype \\
       HQ		& 2		& Integer	& Haplotype quality \\
+      LAA		& .		& Integer	& Strictly increasing indices into REF and ALT, indicating which alternate alleles are relevant (local) for the current sample \\
+      LAD		& .		& Integer	& Read depth for each of the local alternate alleles listed in LAA \\
+      LGT		& .		& String 	& Genotype against the local alleles \\
+      LPL		& .		& Integer	& Phred-scaled genotype likelihoods rounded to the closest integer for genotypes that involve the local alternative alleles listed in LAA \\
```

Comment:
Why are these all number `.`?

We had explicit dimensions before: A, R, G, 1 (GT), etc.  We now have local equivalents for these, but it feels like there should be some better language to explain that rather than just doing a *shrug*.

Particularly LGT which surely is the same dimension as GT:  Type=String, Number=1.  (Although personally I think it was a mistake to shoehorn in multiple data types to the one GT field with an entirely new set of language syntax for one field only)

Isn't LAA type `A` with one entry per alternate allele (or `R` if we went with LA).  Similarly for LAD, although I don't know how we'd represent the depth for REF in a REF/ALT call if it's `A`.  It does feel cleaner to have everything as `R` for logical sense, but maybe some are already and it's only LAA with which `A`?  Not explicitly listing the dimensions isn't aiding clarity of how this is meant to work.

If we don't want to define them as terms in the grammar, then at least explicitly specify the dimensions within the text.  LPL gives no clue at all as to how many values there are.   I understand what's going on, but only because I already understand what's happening...